internal/infrastructure/bot/handlers.go

```diff
@@ -271,14 +272,63 @@ func (b *Bot) PrivateSummary(i *discordgo.InteractionCreate) error {
 		return err
 	}
 
-	err = b.eventHandler.OnPrivateSummary(summary.PrivateSummaryRequest{
-		GuildID: gID,
-		UserID:  uID,
+	isSummaryWithoutRespawns := len(i.ApplicationCommandData().Options) < 1
+	if isSummaryWithoutRespawns {
+		err = b.eventHandler.OnPrivateSummary(summary.PrivateSummaryRequest{
+			GuildID: gID,
+			UserID:  uID,
+		})
+		if err != nil {
+			return err
+		}
+	} else {
+		spotNames := strings.Split(i.ApplicationCommandData().Options[0].StringValue(), ", ")
+		err = b.eventHandler.OnPrivateSummary(summary.PrivateSummaryRequest{
+			GuildID:   gID,
+			UserID:    uID,
+			SpotNames: spotNames,
+		})
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
I think we could avoid big `if` branches here, something along the lines:

```go
var spotNames []string = nil

if(Options >= 1) {
  spotNames = split(...)
}

// make request
```

Reply:
This way we have a single `if` statement just for setting the value of `spotNames` if input was given, but we have a single place where we create a private summary request.